epoxy-preloadersample/src/androidTest/java/com/airbnb/epoxy/preloadersample/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.airbnb.epoxy.preloadersample;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+@RunWith(AndroidJUnit4.class)
```

Comment:
we don't need this specific test - although it would be really nice to have an integration test for the sample